lux/executor/PandasExecutor.py

```diff
@@ -394,6 +394,18 @@ def execute_2D_binning(vis: Vis) -> None:
             x_attr = vis.get_attr_by_channel("x")[0].attribute
             y_attr = vis.get_attr_by_channel("y")[0].attribute
 
+            if vis.data[x_attr].dtype == np.dtype('O'):
+                try:
+                    vis.data[x_attr] = vis.data[x_attr].astype(float)
+                except ValueError:
+                    pass
```

Comment:
No we probably should not raise an exception here because general string columns are also recognized as object type, so we don't want to throw an error in those cases.